docs/integrations/aws-s3/discovery.md

```diff
@@ -26,3 +26,16 @@ catalog:
 ```
 
 Note the `s3-discovery` type, as this is not a regular `url` processor.
+
+As this processor is not one of the default providers, you will also need to add the below:
```

Comment:
Perhaps "... will also need to add it to `packages/backend/src/plugins/catalog.ts`:"?

Reply:
makes sense to clarify location earlier than the code snippet :)